starknet-rpc-test/Cargo.toml

```diff
@@ -10,9 +10,13 @@ anyhow = "1.0.80"
 assert_matches = "1.5.0"
 async-lock = "3.1.0"
 async-trait = { workspace = true }
+blockifier = { workspace = true }
```

Comment:
Not sure we nee all those new crates. Can you give it a look and cleanup the one that we are not using